### DISCUSSION TITLE
Add output formatting to the customize section of every visualization

### DISCUSSION THREAD

**Discussion by vrenvren** [apache/superset#19196] [Ideas]
The visualizations provided by the current version (1.0.1) of Superset could be a more refined, coherent set of tools. Now every visualization has its own style and options. Some basic functionalities should be available for every visualization

In addition to the color schemes, which seem to be the most persistent customization feature, every chart presenting numbers should at a minimum have the option to customize the number format. Controlling the number format at a data source level in some charts and enabling it to be changed in others is odd; it should be enabled in every visualization.

Concretely, providing the "Number format" field, as in pivot table, is what I am hoping to see.

**Comment by poleft** [comment]
Agree! Looking forward to the upgrade !

**Comment by ValentinC-BR** [comment]
Agree ! I think Number formatting is an important area of improvement. It would definitely improve end-users' daily use of Superset.

Users should be able to chose the number format for each metric / numeric dimension displayed in a chart
